README.md

```diff
@@ -1,2 +1,49 @@
 # k4e-scale
-The purpose of this project is to test the the scalability and performance of k4e project using JMeter
+The purpose of this project is to test the scalability and performance of [k4e project](https://github.com/jakub-dzon/k4e-operator) using [JMeter](https://jmeter.apache.org).
+
+## Installation
+The test plan for testing k4e is using the [Parallel Sampling controller](https://github.com/Blazemeter/jmeter-bzm-plugins/blob/master/parallel/Parallel.md) that needs to be installed.
+Instructions for installing plugins can be found on [here](https://jmeter-plugins.org/install/Install/).
+
+## Running the test plan
+Before the test can be run, the test plan needs to be prepared.
+The following parameters need to be set:
+* BEARER_TOKEN - the token is used to access the K8s API (see details below)
```

Comment:
then maybe `K8S_BEARER_TOKEN` or something which doesn't have beaker in it?

Reply:
well, there is no beaker mentioned anywhere in this PR ;-)
but your suggestion for the naming is better. I'll change it.

Reply:
my bad, I read beaker. too many beakers lately.